tasks/awx.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Create directory for AWX
+  file:
+    path: "{{ awx_home }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0750
+
+- name: Clone AWX
+  git:
```

Comment:
I suggest checking out a specific version. That way we avoid introducing breaking changes.

Reply:
I cannot find anything I can use